scripts/ock-once-swtok.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+PIN=${OCK_PIN:-"12345678"}
+
+if id -u | grep -qw 0 && groups | grep -qwv pkcs11 ; then
+    echo "missing pkcs11 group membership"
+    exit 77
+fi
+
+if ! command -v pkcsslotd > /dev/null; then
+    echo "pkcsslotd is required"
```

Comment:
where do I find it?

Reply:
Same as for p11sak, it is part of the RPM. I'll change the message.